server/services/flags-lookup.js

```diff
@@ -12,10 +13,17 @@ export const getFlags = async () => {
     const flagName = flag.flagName;
     const cohortSettings = flag.flagCohortSettings;
     const settings = cohortSettings.reduce((result, item) => {
-      const cohortName = item.featuresCohort.data.cohortName;
-      const cohortSetting = item.flagStatus;
-      result[cohortName] = cohortSetting;
-      return result;
+      if (item.featuresCohort.data) {
+        const cohortName = item.featuresCohort.data.cohortName;
+        const cohortBooleanSetting = item.flagStatus;
+        const cohortValue = RichText.asText(item.flagValue).trim();
+        if (cohortValue.length > 0) {
```

Comment:
Should we not check for empty string? I know `.length` works but it makes it feel like an array.